public/js/controllers/mempool.js

```diff
@@ -73,13 +73,15 @@
             })
             ws.registerEvtHandler("mempool", (evt) => {
                 var m = JSON.parse(evt)
+                console.log("update", evt);
                 this.renderLatestTransactions(m.latest, false)
                 $(this.numTicketTarget).text(m.num_tickets)
                 $(this.numVoteTarget).text(m.num_votes)
                 keyNav(evt, false, true)
                 ws.send("getmempooltxs", "")
             });
             ws.registerEvtHandler("getmempooltxsResp", (evt) => {
+                console.log("resp", evt);
```

Comment:
This logging aren't necessary unless you were using them to test. @chappjc Why do we even log websocket responses on the frontend?

Reply:
Found those comments in the source code I inherited. Will take them all out. Didn't really like seeing them either.

Reply:
You still need to do away with this